index.html

```diff
@@ -48,7 +48,7 @@ <h1>JOSH GLANTZ</h1>
                 <p>Guide to Yosemite Remake</p>
             </a>
         </div>
-        <div>
+        <div class="optional">
```

Comment:
If I am interpreting your CSS right, you are completely hiding these two sections in the smallest view. Is that what you intended? If so, why? In general, you don't want to hide content for mobile; you want all users to see all content (There are some cases where it makes sense though: you may have 2 copies that display differently so one is always hidden, it may be a strictly stylistic thing that doesn't constitute "content", etc).

Reply:
Part of the assignment, I suspect to practice hiding specific content. The mobile wireframe specified that the 4th & 8th img/divs had to dissapear.
I get that I wouldn't normally just drop something like that for mobile.